drivers/store/redis/store.go

```diff
@@ -3,42 +3,81 @@ package redis
 import (
 	"context"
 	"fmt"
+	"strings"
 	"time"
 
 	libredis "github.com/go-redis/redis/v8"
 	"github.com/pkg/errors"
-
```

Comment:
Could you keep the newline here, we prefer to have our import in 3 different group:

- stdlib
- external
- internal

Thank you.